lib/models/project.js

```diff
@@ -281,7 +281,7 @@ class Project {
   /**
     Calls `require` on a given module.
```

Comment:
good point, lets make sure the docs are more fleshed out before flipping to public.

* describe what this does
* describe when one may want to use this.